lib/build-cloud/securitygroup.rb

```diff
@@ -80,21 +80,35 @@ def rationalise_rules( authorized_ranges )
         
         security_group.ip_permissions.each do |r|
 
-            c = {
-                :min_port    => r['fromPort'],
-                :max_port    => r['toPort'],
-                :ip_protocol => r['ipProtocol'],
-            }
-
             if r['groups'] != []
```

Comment:
Not your change, and only a stylistic quibble, not an issue...  but may be more legible to use `unless r['groups'].empty?` here